assets/javascripts/initializers/code-bytes.js

```diff
@@ -175,8 +176,36 @@ function initializeCodeByte(api) {
         div.appendChild(saveButton);
       }
     });
-  }),
-    { id: 'codebyte-preview' };
+  },
+  { id: 'codebyte-preview' });
+
+  api.modifyClass("controller:composer", {
+    save(...args) {
```

Comment:
oh wow it's good to know we can do this. def out of scope for this PR but could we kill the "save to post" button if we move that logic here?

Reply:
Oh yeah! I'll write that up. https://codecademy.atlassian.net/browse/REACH-1066